### DISCUSSION TITLE
Cannon

### DISCUSSION THREAD

**Discussion by IRobot1** [nartc/angular-three#55] [Q&A] [answered]
Using the [Cannon introduction](https://angular-three.netlify.app/?path=/docs/cannon-introduction--page) as an example, if I scaled the cube to 50% [scale]="0.5", how do I recompute the bounding box?

At 50%, the box appears above the floor

![image](https://user-images.githubusercontent.com/25032599/154819796-b314285e-df43-4bfc-a611-8af3ee6a19dd.png)

**Comment by nartc** [comment, marked as answer]
@IRobot1 I've updated the introduction guide to show how to use `scale`. Check it out. Thank you for asking.
